proof-systems/src/groth16/r1cs_to_qap.rs

```diff
@@ -213,7 +213,10 @@ impl R1CStoQAP {
         domain.coset_ifft_in_place(&mut ab);
 
         // we drop the leading coefficient, as deg(h(Z)) = n-2.
-        assert!(ab.pop().unwrap() == zero);
+        // TODO: check error message
```

Comment:
Let's leave the assertion: if it fails it means that there is a bug in the software, no one can trigger this panic for a DoS attack